src/logic/currencyValues/store/actions/fetchCurrencySelectedValue.js

```diff
@@ -7,15 +7,16 @@ import { AVAILABLE_CURRENCIES } from '~/logic/currencyValues/store/model/currenc
 import type { GlobalState } from '~/store'
 
 // eslint-disable-next-line max-len
-const fetchCurrencySelectedValue = (currencyValueSelected: $Keys<typeof AVAILABLE_CURRENCIES>) => async (
-  dispatch: ReduxDispatch<GlobalState>,
-) => {
+const fetchCurrencySelectedValue = (
+  safeAddress: string,
+  currencyValueSelected: $Keys<typeof AVAILABLE_CURRENCIES>,
```

Comment:
the same rule applies to this param: `currencyValueSelected` > `selectedCurrency`

Reply:
Fixed